Tasks/InstallAppleProvisioningProfile/task.json

```diff
@@ -26,34 +26,12 @@
     "instanceNameFormat": "Install an apple provisioning profile",
     "inputs": [
         {
-            "name": "provProfileSource",
```

Comment:
This doesn't work well since users still need a new agent and the logic will have to be split between pre execution and execution since pre execution is run before Get sources.